common/types/firestore-types.ts

```diff
@@ -6,6 +6,11 @@ export type FirestoreCommon = {
   readonly order: number;
 };
 
+export type FirestoreCommonNew = {
+  readonly owner: string[];
```

Comment:
use `readonly string[]` for immutability

Reply:
> BTW, this will probably leave unmerged until a frontend code change is done in another PR.

sounds good. I thought it might be easier to fix things once the schema was already changed, but this is definitely the safer option.